crates/subspace-farmer/src/bin/subspace-farmer/commands/farm/metrics.rs

```diff
@@ -286,4 +331,56 @@ impl FarmerMetrics {
             )])
             .observe(time.as_secs_f64());
     }
+
+    pub(super) fn update_farm_size(
+        &self,
+        single_disk_farm_id: &SingleDiskFarmId,
+        sectors: i64,
```

Comment:
```suggestion
        sectors: SectorIndex,
```

And if you need to convert it to bigger integer prefer `u64::from(sectors)`, this way if some of the types change in incompatible way it'll stop compiling, otherwise `as u64` will potentially truncate the data. Just a good practice.

Reply:
Just updated taking into account all the recommendations